tests/test_run_app.py

```diff
@@ -685,6 +686,24 @@ async def on_startup(app):
     exc_handler.assert_called_with(patched_loop, msg)
 
 
+def test_run_app_keepalive_timeout(patched_loop):
```

Comment:
It's usually better to use the [`mocker`](https://pypi.org/project/pytest-mock/#usage) fixture from `pytest-mock` (it's already integrated as a test dependency) for accessing the `MagicMock` class.

Reply:
Sure, I can update it. What exactly is the benefit of using the fixture?

Reply:
Just the consistency: if the lib is integrated, the expectation is that it would be used. Also, pytest design is more fixture-driven so external things seem foreign.

Reply:
Got it. I can change this (and the rest of the comments) if @asvetlov agrees with the feature itself.